docs/api-testing/quickstart.md

```diff
@@ -7,94 +7,89 @@ description: Learn how to quickly generate an API test using the payload from an
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This is a quickstart guide for getting up and running with a Sauce Labs API Test using the **HTTP Client** or **Import from Postman/Spec File** feature. Let's get started!
+This is a quickstart guide for getting up and running with a Sauce Labs API Test using the **HTTP Client** feature. Let's get started!
 
 
 
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
-* If your APIs are behind a firewall, you'll need to set up a [Sauce Connect Proxy](/secure-connections/sauce-connect/) tunnel before proceeding.
 
+:::note
+If your APIs are behind a firewall, you'll need to set up a [Sauce Connect Proxy](/secure-connections/sauce-connect/) tunnel before proceeding.
+:::
 
-## Create Project
-
-1. Log in to Sauce Labs, then click **API Testing** > **Get Started**.<br/>
-  <img src={useBaseUrl('img/api-fortress/2021/09/landingPage.png')} alt="API Testing landing page" width="600" />
-
-2. To set up a test, you'll need a Project first. On the API Testing dashboard, click the **Create Project** button.<br/>
-  <img src={useBaseUrl('img/api-fortress/2021/01/createProject.png')} alt="Create a Project UI" width="500" />
-
-3. In the **New Project** window, enter your **Project Name**. Optionally, you can also add a **Description** and **Notes**). Click **Save** when you're finished. <br/>
-  <img src={useBaseUrl('img/api-fortress/2021/01/newProject.png')} alt="New Project window" width="400" />
 
-  This will bring you to the **Tests** dashboard under the project you've just created.
+## Create Project
 
+1. Log in to Sauce Labs, then click **API Testing**.
+2. To start testing, you'll need a Project first. On the API Testing dashboard, click the **Create Project** button.<br/><img src={useBaseUrl('img/api-fortress/2021/01/createProject.png')} alt="Create Project" width="450" />
+3. In the **Create a New Project** window:
+   * Set the **Create from** dropdown to **blank project**.
+   * Enter your **Project Name**.
+   * Optionally, you can add **Tags**, a **Description**, and/or **Notes**.
+   * Set the **Access** dropdown to **Organization**.
```

Comment:
@kimsauce Why user should set to "Organization"? It's up to user to choose what the level should be. If user is the only one in the company "Organization" is the only one available so he don't have to set up anything, otherwise they can choose what they want